src/index.html

```diff
@@ -11,6 +11,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="index.html"
+      method="post"
+      class="big__bar"
+      data-qa="big"
+    >
+      <img
+        src="images/Search.svg"
+        alt="search"
+        class="big_search"
+      >
+      <label for="big_search_input"></label>
```

Comment:
you don't need this label

Reply:
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/91217030/f1df9662-a9c6-4d68-9537-ffd31787771f)
when i delet label , i get this